### PR TITLE
Fix up tune control docs

### PR DESCRIPTION
**Pull request by hamishwillee**
Addresses #10610

This provides fix for tune_control docs statement that there are 15 default tunes.

To fix this the docs now refer to system tunes (rather than default tunes) and don't mention the number of tunes.

### CODE REVIEW

**Comment by hamishwillee** [comment]
@bkueng I was wondering if we should perhaps take the time to add tune_control to module documentation. Something like this:
```
	PRINT_MODULE_DESCRIPTION(
		R"DESCR_STR(
### Description
External tune control for testing.

Information about the tune format and predfined system tunes can be found here:
https://github.com/PX4/Firmware/blob/master/src/lib/tunes/tune_definition.desc

### Examples

tune_control play -t 2

)DESCR_STR");


	PRINT_MODULE_USAGE_NAME("tune_control", "system");
	PRINT_MODULE_USAGE_COMMAND_DESCR("play","Play system tune, tone, or melody");
        PRINT_MODULE_USAGE_PARAM_INT('t', 1, 1, 21, "Play predefined system tune", true);
        PRINT_MODULE_USAGE_PARAM_INT('f', 0, 0, 22, "Frequency of tone in Hz (0-22kHz)", true);
	PRINT_MODULE_USAGE_PARAM_INT('d', 1, 1, 21, "Duration of tone in us", true);
	PRINT_MODULE_USAGE_PARAM_INT('s', 40, 0, 100, "Strength of tone (0-100)", true);
	PRINT_MODULE_USAGE_PARAM_STRING('m', nullptr, "<string> - e.g. \"MFT200e8a8a\"",
					 "Melody in string form", true);
	PRINT_MODULE_USAGE_COMMAND("libtest");
	PRINT_MODULE_USAGE_COMMAND_DESCR("stop","Stop playback (use for repeated tunes)");
```

Questions 
- what is `libtest` argument?
- Is there a way to indicate "there is no default" (at least in docs)? For example, the frequency above must be defined if a tone is defined. There is no default value though, so would be best this was not shown in docs.
- Any reason not to include the px4_module.h which defines these doc features?

**Comment by bkueng** [comment]
Thanks @hamishwillee, your suggestion sounds good.

> what is libtest argument?

Not quite sure what it does but it's there to test the tunes library.


> Is there a way to indicate "there is no default" (at least in docs)? For example, the frequency above must be defined if a tone is defined. There is no default value though, so would be best this was not shown in docs.

no, you can write this in the description. There's only the optional boolean flag (last argument).

> Any reason not to include the px4_module.h which defines these doc features?

Yes you're going to need that when using the `PRINT_MODULE_USAGE_NAME` methods.

**Comment by hamishwillee** [comment]
OK, I have added this as module docs. Couldn't get libtest to do anything so just put in placeholder text. @bkueng Feel free to take this and modify to whatever state you think it needs :-)

The current version renders out .md as shown below (and the console output in simulator below that)

----

## tune_control
Source: [systemcmds/tune_control](https://github.com/PX4/Firmware/tree/master/src/systemcmds/tune_control)


### Description

Command-line tool to control & test the (external) tunes.

Tunes are used to provide audible notification and warnings (e.g. when the system arms, gets position lock, etc.).
The tool requires that a driver is running that can handle the tune_control uorb topic.

Information about the tune format and predefined system tunes can be found here:
https://github.com/PX4/Firmware/blob/master/src/lib/tunes/tune_definition.desc

### Examples

Play system tune #2:

    tune_control play -t 2



### Usage {#tune_control_usage}
```
tune_control <command> [arguments...]
 Commands:
   play          Play system tune, tone, or melody
     [-t <val>]  Play predefined system tune
                 default: 1
     [-f <val>]  Frequency of tone in Hz (0-22kHz)
                 default: 0
     [-d <val>]  Duration of tone in us
                 default: 1
     [-s <val>]  Strength of tone (0-100)
                 default: 40
     [-m <val>]  Melody in string form
                 values: <string> - e.g. "MFT200e8a8a"

   libtest       Test library

   stop          Stop playback (use for repeated tunes)
```

----


image

File: src/systemcmds/tune_control/tune_control.cpp [hunk 1]
```diff
@@ -62,18 +64,40 @@ extern "C" {
...
+
+Information about the tune format and predefined system tunes can be found here:
+https://github.com/PX4/Firmware/blob/master/src/lib/tunes/tune_definition.desc
+
+### Examples
+
+Play system tune #2:
+
+    tune_control play -t 2
+
```
Comment (hamishwillee) [inline src/systemcmds/tune_control/tune_control.cpp]: might be a bit much white space here?

Comment (bkueng) [reply src/systemcmds/tune_control/tune_control.cpp]: I typically do:
Play system tune #2:
$ tune_control play -t 2
Which then will be converted to use the backticks for markdown output.

Comment (hamishwillee) [reply src/systemcmds/tune_control/tune_control.cpp]: Thanks. Done! Merge if you're happy.

**Comment by dagar** [comment]
There's a CI failure in this PR, which is still now present in master.

image

**Comment by hamishwillee** [comment]
Thanks @dagar . Fix in https://github.com/PX4/Firmware/pull/10675

**Pull request merged**